src/vm/arm64/cgencpu.h

```diff
@@ -153,11 +153,29 @@ inline TADDR GetSP(const T_CONTEXT * context) {
     return TADDR(context->Sp);
 }
 
-inline PCODE GetLR(const T_CONTEXT * context) {
+inline TADDR GetLR(const T_CONTEXT * context) {
     LIMITED_METHOD_DAC_CONTRACT;
     return PCODE(context->Lr);
```

Comment:
you can remove PCODE() it is not required for arm64. It is confusing right now because return type is TADDR and you are casting to PCODE